src/installer/installer.go

```diff
@@ -234,6 +268,29 @@ func (i *installer) extractIgnitionToFS(ignitionPath string) (err error) {
 	return err
 }
 
+func (i *installer) patchMastersIgnition(yamlFile, fieldName, newData, op string, yamlStruct interface{}) error {
```

Comment:
I think this is sufficiently complex to warrant a test.

Reply:
Agree, will do it (this is why it is still [WIP])
:)